src/main/webapp/app/tasks/dialog/task-due-date-dialog/task-due-date-dialog.component.html

```diff
@@ -0,0 +1,16 @@
+
```

Comment:
Why did you decide to do a dialog time picker and an inline picker?

Reply:
Personally I like inline picker , I don't know which approach you guys follow in Coyo , so I Implemented both. It could be a design decision.